src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -26,6 +26,23 @@
   --pf-c-context-selector__toggle--m-plain--m-text--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-context-selector__toggle--m-plain--m-text--PaddingLeft: var(--pf-global--spacer--sm);
 
+  // ::before
+  --pf-c-context-selector__toggle--before--BorderWidth: var(--pf-c-context-selector__toggle--BorderWidth); // update value at breaking change
+  --pf-c-context-selector__toggle--before--temp--BorderBottomColor: var(--pf-global--BorderColor--300); // change to --pf-c-context-selector__toggle--before--BorderColor at breaking change
+  --pf-c-context-selector__toggle--before--BorderTopColor: var(--pf-c-context-selector__toggle--BorderTopColor); // remove at breaking change
+  --pf-c-context-selector__toggle--before--BorderRightColor: var(--pf-c-context-selector__toggle--BorderRightColor); // remove at breaking change
+  --pf-c-context-selector__toggle--before--BorderLeftColor: var(--pf-c-context-selector__toggle--BorderLeftColor); // remove at breaking change
+
+  // ::after
+  --pf-c-context-selector__toggle--after--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
Should you use `--pf-c-context-selector__toggle--before--BorderWidth` as the value here since the border width was previously controlled by `--pf-c-context-selector__toggle--before--BorderWidth`? If a user updated that to `4px`, it would update all 4 border widths. If we merge the updates in this PR, their bottom border width is no longer 4px.

Just to show it, here's that override in the current context selector - https://codepen.io/mcoker/pen/oNWgjZL

And here's the CSS from this PR with that override (the commented line is my suggestion) - https://codepen.io/mcoker/pen/wvdBKoy